addons/binding/org.openhab.binding.homie/build.properties

```diff
@@ -0,0 +1,8 @@
+source.. = src/main/java/
+output.. = target/classes
+bin.includes = META-INF/,\
+               .,\
+               OSGI-INF/,\
+               ESH-INF/,\
+               lib/org.eclipse.paho.client.mqttv3-1.1.0.jar,\
```

Comment:
@aisebastian This service is not used in OH2, because there are no Mqtt related bindings ported to OH2. Did you see my response to your question in the eclipse forum? Look at the PRs of the eclipse smarthome repository and you will find some API consumers :)